lib/+iDynTree2WBTmappers/KinDynComputations.m

```diff
@@ -0,0 +1,73 @@
+classdef KinDynComputations < handle
+    %KINDYNCOMPUTATIONS The calss wraps the iDynTree::KinDynComputations methods using Simulink function calls.
+    %   Detailed explanation goes here
+    
+    properties (Access = private)
+        w_H_b (4, 4) double;;
+        s;
+        base_pose_dot (6, 1) double;;
+        s_dot;
+        NDOF double;
+    end
+    
+    methods
+        function obj = KinDynComputations()
+        end
+        
+        function setRobotState(obj,w_H_b,s,base_pose_dot,s_dot)
+            % Sets the robot state the wrapper methods depend on
+            [obj.w_H_b,obj.s,obj.base_pose_dot,obj.s_dot] = deal(w_H_b,s,base_pose_dot,s_dot);
+            obj.NDOF = numel(s_dot);
+        end
+        
+        function [w_H_b,s,base_pose_dot,s_dot,NDOF] = getRobotState(obj)
+            % Gets the robot state the wrapper methods depend on
+            [w_H_b,s,base_pose_dot,s_dot,NDOF] = deal(obj.w_H_b,obj.s,obj.base_pose_dot,obj.s_dot,obj.NDOF);
+        end
+        
+        function [ack,J_LRfoot] = getFrameFreeFloatingJacobianLRfoot(obj,LRfoot)
+            ack = true;
+            switch LRfoot
+                case 'LFoot'
+                    J_LRfoot = simFunc_getFrameFreeFloatingJacobianLFoot(obj.w_H_b,obj.s);
+                case 'RFoot'
+                    J_LRfoot = simFunc_getFrameFreeFloatingJacobianRFoot(obj.w_H_b,obj.s);
+                otherwise
+                    error('Unsupported "getFrameFreeFloatingJacobian" input parameter.');
+                    ack = false;
```

Comment:
Not sure about this. If the error occurs, the program should exit and the ack will be not changed, isn't it?

Reply:
Absolutely. There was actually a warning. I kept the `ack` variable for conserving the previous interface of the usd `KinDynComputations` methods. I had in mind that that we might want to get back to using the bindings if some day we manage to generate C++ code from that. For that reason I created a `KinDynComputations` class inside a new package `matlab-whole-body-simulators/lib/+iDynTree2WBTmappers`.

For the sake of coherence, I removed the call to `error` in this method, as the `ack` is checked by the caller, which triggers an error on its own.

Reply:
Fixed in the next commit.